src/components/NcActions/NcActions.vue

```diff
@@ -1471,10 +1474,17 @@ export default {
 		/**
 		 * Called when popover is shown after the show delay
 		 */
-		onOpen() {
+		onOpened() {
 			this.$nextTick(() => {
 				this.focusFirstAction(null)
 				this.resizePopover()
+
+				/**
+				 * Event emitted when the popover menu is opened.
+				 *
+				 * This event is emitted after `update:open` was emitted and the opening transition finished.
+				 */
+				this.$emit('opened')
```

Comment:
Feature not fix (new event)

Reply:
Fix for NcPopover, feat for NcActions

Reply:
but the fix in ncpopover is unrelated to the feature for NcActions.
So this is semantically a feature PR ;)

Reply:
Yeah, was missing :shrug: